### ISSUE TITLE
Copy not including full content

### ISSUE THREAD

**Issue by asio** [ajaxorg/ace#4702]
### Describe the bug

Browser: Firefox version 102.0a1
Url: https://ace.c9.io/

When selecting the full code sample on the https://ace.c9.io/ and copying with ctrl+c then in the same code editor hitting ctrl+v I only get the last line pasted.

### Expected Behavior

When selecting all content and ctrl+c then ctrl+v, I expect the following content:

```
/**
 * In fact, you're looking at ACE right now. Go ahead and play with it!
 *
 * We are currently showing off the JavaScript mode. ACE has support for 45
 * language modes and 24 color themes!
 */

function add(x, y) {
    var resultString = "Hello, ACE! The result of your math is: ";
    var result = x + y;
    return resultString + result;
}

var addResult = add(3, 2);
console.log(addResult);
```

### Current Behavior

I get the following content:

```
console.log(addResult);
```

### Reproduction Steps

1. Go to https://ace.c9.io/
2. Select all code in the editor
3. Press ctrl+c on keyboard
4. While all code still selected press ctrl+v

### Ace Version

Unkown

**Comment by andrewnester** [comment]
Tried to reproduce it on FF Nightly build (102.0a1) 64-bit but wasn't able to. OS: macOS

Could you provide what OS are you using?

**Comment by asio** [comment]
OS: Windows 10 Enterprise
Version: 21H2
Build: 19044.1706

**Comment by nightwing** [comment]
This can happen if `dom.event.clipboardevents.enabled`  preference is set to false https://blog.raw.pm/en/disable-clipboard-events could you check if that is the case.

**Comment by asio** [comment]
> This can happen if `dom.event.clipboardevents.enabled` preference is set to false https://blog.raw.pm/en/disable-clipboard-events could you check if that is the case.

Yes you were correct. I changed it from false to true and now it works

**Comment by nightwing** [comment]
This can happen if dom.event.clipboardevents.enabled  preferrence is set to
false https://blog.raw.pm/en/disable-clipboard-events could you check if
that is the case.
